tools/inspect/Inspect.cpp

```diff
@@ -120,10 +120,10 @@ bool writer_count = false;
 
 void print_usage(bool for_error = false)
 {
-  std::ostream& os = for_error ? std::cout : std::cerr;
+  std::ostream& os = for_error ? std::cerr : std::cout;
```

Comment:
I'm not sure what was intended originally. @ClaytonCalabrese ?

Reply:
I wrote this part as part of #3189. Usage can be part of help, which is not an error, or part of a error for invalid arguments. I noticed I mixed this up.